From: WANG Cong <xiyou.wangcong@gmail.com>
Date: Sat, 13 Aug 2016 22:34:58 -0700
Subject: net_sched: fix a typo in tc_for_each_action()
Patch-mainline: v4.8-rc3
Git-commit: 0c23c3e705691cfb99c94f2760df2b456fe45194
References: bsc#966170 FATE#320225 bsc#966172 FATE#320226

It is harmless because all users pass 'a' to this macro.

Fixes: 00175aec941e ("net/sched: Macro instead of CONFIG_NET_CLS_ACT ifdef")
Cc: Amir Vadai <amir@vadai.me>
Signed-off-by: Cong Wang <xiyou.wangcong@gmail.com>
Acked-by: Jamal Hadi Salim <jhs@mojatatu.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 include/net/act_api.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/net/act_api.h
+++ b/include/net/act_api.h
@@ -145,7 +145,7 @@ int tcf_action_copy_stats(struct sk_buff
 	(list_empty(&(_exts)->actions))
 
 #define tc_for_each_action(_a, _exts) \
-	list_for_each_entry(a, &(_exts)->actions, list)
+	list_for_each_entry(_a, &(_exts)->actions, list)
 #else /* CONFIG_NET_CLS_ACT */
 
 #define tc_no_actions(_exts) true
